engine/apps/slack/scenarios/shift_swap_requests.py

```diff
@@ -41,9 +40,7 @@ def _generate_blocks(self, shift_swap_request: "ShiftSwapRequest") -> Block.AnyB
                     "type": "section",
                     "text": {
                         "type": "mrkdwn",
-                        # TODO: I believe it'll be easier to wait to generate this until we have the schedule override changes in place
-                        # NOTE: use apps.slack.utils.format_datetime_to_slack method to format the datetimes
-                        "text": "*📅 Shift Details*: 9h00 - 17h00 (UTC) daily from Monday July 24, 2023 - July 28, 2023",
+                        "text": f"*📅 Shift Details*: {shift_swap_request.shifts_summary}",
```

Comment:
another approach here could be not to simply output this as a single string and instead loop over `shift_swap_request.shifts()` and output each schedule event as a separate line in the Slack message like this:
<img width="473" alt="Screenshot 2023-08-01 at 12 54 46" src="https://github.com/grafana/oncall/assets/9406895/a10236af-4be8-4690-a7cd-27cd7d935398">